doc/meson.build

```diff
@@ -36,3 +36,22 @@ if get_option('manpages')
     install_dir: [join_paths(get_option('mandir'), 'man1'), join_paths(get_option('mandir'), 'man5')],
   )
 endif
+
+doxygen = find_program('doxygen', required: false)
```

Comment:
Why look up doxygen if the option is disabled?

Reply:
Good point -- fixed.